### ISSUE TITLE
[BUG]: `pipeline requires workspace with name "model-workspace" be provided by pipelinerun`

### ISSUE THREAD

**Issue by adelton** [opendatahub-io/ai-edge#190]
### Describe the bug

Following the `pipelines/README.md`, the build-container-image-tensorflow-housing-t2mtr PipelineRun fails with

```
PipelineRun model-to-edge/build-container-image-tensorflow-housing-t2mtr doesn't bind Pipeline model-to-edge/build-container-image's Workspaces correctly: pipeline requires workspace with name "model-workspace" be provided by pipelinerun
```

### To Reproduce

1. Get a fresh OCP cluster.
2. As admin, install the Red Hat OpenShift Pipelines 1.13.0 operator via a web console.
3. Per `pipelines/README.md`, as non-admin user, `oc new-project model-to-edge`.
4. Per `pipelines/README.md`, edit credentials to the S3:
    ````
    cp tekton/build-container-image-pipeline/aws-env.yaml tekton/build-container-image-pipeline/aws-env-real.yaml
    vi tekton/build-container-image-pipeline/aws-env-real.yaml
    ````
5. Per `pipelines/README.md`, as non-admin user, `oc create -f tekton/build-container-image-pipeline/aws-env-real.yaml`.
6. Per `pipelines/README.md`, update `aws-bucket-name` value in both `pipelines/tekton/build-container-image-pipeline/build-container-image-pipelinerun-bike-rentals.yaml` and `pipelines/tekton/build-container-image-pipeline/build-container-image-pipelinerun-tensorflow-housing.yaml` to point to an S3 bucket with copies of the models.
7. Per `pipelines/README.md`, as non-admin user,
    ````
    oc apply -k tekton/build-container-image-pipeline/
    oc create -f tekton/build-container-image-pipeline/build-container-image-pipelinerun-bike-rentals.yaml
    # and / or
    oc create -f tekton/build-container-image-pipeline/build-container-image-pipelinerun-tensorflow-housing.yaml
    ````
8. As admin on the web console, check the Pipelines and PipelineRuns in the **model-to-edge** namespace.

See the build-container-image-bike-rentals-9vw2d PipelineRun eventually reported as Succeeded.

See the build-container-image-tensorflow-housing-t2mtr reported as Failed, with a log snippet message
> PipelineRun model-to-edge/build-container-image-tensorflow-housing-t2mtr doesn't bind Pipeline model-to-edge/build-container-image's Workspaces correctly: pipeline requires workspace with name "model-workspace" be provided by pipelinerun

### Expected behavior

Both PipelineRuns (build-container-image-bike-rentals-9vw2d and build-container-image-tensorflow-housing-t2mtr) end up in the Succeeded state.

### Screenshots (if applicable)

![Screenshot_2023-12-13_10-15-45](https://github.com/opendatahub-io/ai-edge/assets/2536912/f2c0ecea-20af-41e9-94d9-7b51905bfc20)

**Comment by adelton** [comment]
@jackdelahunt, @LaVLaS, since the `model-workspace` was introduced via https://github.com/opendatahub-io/ai-edge/pull/115, it seems that PR might be the source of the regression. When I reset my repo to 63ef2dcbd3a0e82afe81f0606abf53364ba7a99b (before https://github.com/opendatahub-io/ai-edge/pull/115 got merged), both the PipelineRuns work as expected (unless one hits https://github.com/opendatahub-io/ai-edge/issues/175 as well).

**Comment by adelton** [comment]
Hello @MarianMacik, was this bug addressed by the 77be5caa034379546c265eaa19a542101f789ea3 commit of https://github.com/opendatahub-io/ai-edge/pull/191, and thus we could mark this issue as fixed?

**Comment by MarianMacik** [comment]
@adelton Sorry, I have just noticed it now. Yes, this bug was addressed. Fixed by #191 in https://github.com/opendatahub-io/ai-edge/commit/77be5caa034379546c265eaa19a542101f789ea3.
